sh.exe.stackdump

```diff
@@ -0,0 +1,4 @@
+Stack trace:
```

Comment:
Hmm - it looks like it's been removed at some point from the main repo - so all goo